### DISCUSSION TITLE
安装ClamAV服务问题

### DISCUSSION THREAD

**Discussion by git-Ander** [1Panel-dev/1Panel#5777] [Q&A]
### 1Panel 版本

v1.10.12-beta.1

### 问题描述

按文档说明安装病毒扫描软件  报 Failed to start clamd@scan.service: Unit not found.

### 重现步骤

主机名称 liunx
发行版本 centos
内核版本 3.10.0-1160.119.1.el7.x86_64
系统类型 x86_64

**Comment by wanghe-fit2cloud** [comment]
> 发行版本 centos

`cat /etc/redhat-release` 命令查看一下具体版本，我们尝试在本地环境复现一下问题。

**Comment by git-Ander** [comment]
CentOS Linux release 7.9.2009 (Core)

**Comment by wanghe-fit2cloud** [comment]
刚刚查阅 ClamAV 相关的官方文档，发现了相关问题，链接如下：
https://github.com/Cisco-Talos/clamav/issues/946

也可以参考该文档，看是否能解决上述问题：
https://www.plesk.com/kb/support/clamav-antivirus-error-in-plesk-failed-to-start-clamdscan-service-unit-not-found/
